config/webpack.config.dev.js

```diff
@@ -282,9 +282,7 @@ module.exports = [{
         include: [
           paths.marketplaceSrc,
           paths.sharedComponents,
-          paths.appNodeModules + '/@gov.au/footer',
-          paths.appNodeModules + '/@gov.au/page-alerts',
-          paths.appNodeModules + '/@gov.au/control-input',
+          paths.appNodeModules + '/@gov.au'
```

Comment:
you don't want to mess with webpack config

Reply:
This prevents ui-kit from overwriting the styles in the header.

Reply:
The new uikit has been merged now so perhaps rebase from master?

Reply:
this will probably be a merge conflict when rebased from master, but I think this approach is better as it means we don't have to modify the webpack config every time we add a new uikit component to the code base.

FYI the `include` is a bit misleading. It doesn't mean webpack will include this path, it is just used in conjunction with `test` to match imports to a particular loader (in this case, `babel`). So I don't believe there is any efficiency found in specifying uikit components, vs just matching on the entire `@gov.au` npm scoped packages.

https://webpack.js.org/configuration/module/#condition (we're very far behind in webpack version but I believe this doco is accurate for our version too)